src/modules/priorities/application/use-cases/__tests__/swap-vm-priorities.use-case.spec.ts

```
@@ -0,0 +1,376 @@
+import { Repository, DataSource, EntityManager } from 'typeorm';
+import { getRepositoryToken } from '@nestjs/typeorm';
+import { NotFoundException, ForbiddenException } from '@nestjs/common';
+import { SwapVmPrioritiesUseCase } from '../swap-vm-priorities.use-case';
+import { Vm } from '../../../../vms/domain/entities/vm.entity';
+import { GetUserVmPermissionsUseCase } from '../../../../permissions/application/use-cases/permission-vm';
+import { LogHistoryUseCase } from '../../../../history/application/use-cases';
+import { PermissionBit } from '../../../../permissions/domain/value-objects/permission-bit.enum';
+
+describe('SwapVmPrioritiesUseCase', () => {
+  let useCase: SwapVmPrioritiesUseCase;
+  let vmRepository: jest.Mocked<Repository<Vm>>;
```

ESLint

[error] 'vmRepository' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)